app/main.py

```diff
@@ -14,4 +14,11 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    count = 0
+
+    for i in range(len(phrase.lower())):
+        if phrase[i].lower() == letter.lower():
+            count += 1
+
+    return count
```

Comment:
Always add a blank line at the end of the file

Reply:
Strange, in PyCharm I have a blank line at the end. Could you recommend how to make sure that it will exist in Github?

Reply:
My bad! You have everything fine with the last line. Thank you for your answer)